LICENSES/curl.txt

```diff
@@ -0,0 +1,22 @@
+COPYRIGHT AND PERMISSION NOTICE
+
+Copyright (C) Daniel Stenberg, <daniel@haxx.se>, and many
+contributors, see the THANKS file.
```

Comment:
The `THANKS` file is `docs/AUTHORS` in libssh2. Perhaps renaming it to `THANKS` would be the best?

Reply:
Me oops. I missed that this is the curl license. This is no biggie, or we might also change it to an URL.

..but, does libssh2 uses the curl license?